src/main/scala/tile/Core.scala

```diff
@@ -41,6 +41,7 @@ trait CoreParams {
   val nL2TLBEntries: Int
   val mtvecInit: Option[BigInt]
   val mtvecWritable: Boolean
+  val clockGate: Boolean = false
```

Comment:
Don't put a default value in a trait, then all users will have to `override` it. Here it should be abstract, and sub-classes of this trait should fill in the value.

Reply:
Oh, I didn't know this rule. will remove default value.